README.md

```diff
@@ -34,32 +34,64 @@ def handler(event:, context:)
     JSON.generate(body)
 end
 
-SignalFx::Lambda::Tracing.register_handler(&method(:handler))
+SignalFx::Lambda.register_handler(metrics: true, tracing: true, &method(:handler))
 ```
 
 `register_handler` will accept any block.
 
+If passing in a block parameter, it must be the last argument.
+
+It also takes these optional arguments:
+- `metrics`: Enable reporting of metrics. Default: `true`
+- `tracing`: Enable tracing. Default: `true`
+
+### Endpoint configuration
+
+If metrics and traces should be reported to a common endpoint, as is the case
+when using the Smart Gateway, a single variable can be used:
+
+```
+SIGNALFX_ENDPOINT_URL = <gateway_url>
```

Comment:
I would take out the spacing around ` = ` because a lot of shells don't allow it and it is also a very strong convention.

Reply:
Also it isn't clear that this is an envvar even though the code uses it as such.